### ISSUE TITLE
[chrX:125899652-125899715][INS][SVLEN=63bp]

### ISSUE THREAD

**Issue by jzook** [jzook/testHG008curation#301]
`chrX	125899652	Minda_155	N	<INS>	.	PASS	SVLEN=63;SVTYPE=INS;SUPP_VEC=ONT_severus_INS22084,ONT_Sniffles2.INS.313M16,ONT_i_414,ONT_ID_50518_1,PB_severus_INS19445,PB_Sniffles2.INS.2F0M16,PB_i_950,PB_ID_51051_1
`

https://v2.genomeribbon.com/?session=https://42basepairs.com/download/s3/giab-data/ribbon-json/ribbon-hg008-cov-bedpe.json&locus=chrX:125899652#ribbon

**Comment by jo-mc** [comment]
Sequence:
ATTTTTTTTTTTTTTTTTTTTTTTTTATTCTTCTTTGTTTTATTTTTCTTTTTTTTTTTTTTTTTT
with varying T counts, after A at start and at end. Deletion of 4bp at insertions site TAAG.

Unknown.

Looks clonal. (see igv and snps to right align to read without insertion) 
Coverage ~160? maybe just lower than 200 here.  chromosome pairs about even coverage.

Possible subclonal deletion of 121 bp (5 reads)  at 124213876. (~2500 bp to right)


![image](https://github.com/user-attachments/assets/969edb0f-7a31-41a5-8d7c-7a7edc9c3557)

![image](https://github.com/user-attachments/assets/422e8a0f-e1ce-4dc3-a6a5-77e609b44f66)

IGV:
![image](https://github.com/user-attachments/assets/cd1f4bc3-6dcb-4b37-859b-85ad9922aa4c)

**Comment by jo-mc** [comment]
@jzook  After looking at the draft vcf, I had another look at the variation in length (which I noted above). Do we need to capture this in vcf (??not sure how).

Here are all the insertions pulled  out and aligned: The core insertion is to right and variation in "T" length on the left:

![image](https://github.com/user-attachments/assets/b6cfe30c-507f-44f3-b86a-37934890b303)

**Comment by jzook** [comment]
Yeah, at some point I think we should try to capture this variation in length since based on the Revio data above it looks like there are at least 2 populations in polyA length and possibly some variation even within each population. It's hard to tell exactly what is happening because the long read data is noisy in homopolymers, but I'm pretty sure at least the two populations are real variation. There's some evidence for these two populations in element and onso as well in the clipped reads, though I couldn't find many reads spanning the insertion. If we're eventually able to see that different subclones have these two different lengths, we could potentially include both alleles in the vcf, but that'll be hard so for now we should look into ways of representing the uncertainty in length

**Comment by jo-mc** [comment]
I'm not so familiar with the  vcf spec, but looking at vcf4.5 something like :

chrA 14 . T <INS> . . IMPRECISE;SVLEN=100;CILEN=-50,50;CIPOS=-10,10 GT 0/1

where imprecise and CILEN are used. CILEN confidence interval of insert size. I'm not sure how to interpret the two values as % or size in bp offset (+ or -)  from specified length. 

Otherwise, using a pedigree ID, as specified for clonal relationships. I guess we need to know that tools reading the VCF can interpret these formats.

**Comment by jo-mc** [comment]
I noticed this was marked as INS:ME so I had another look:

Blat has several hits of this sequence but there is a full length L1Hs close to blat results here:
chr7   +    96846590  96846649
indicating  the this is the source of this insertion.

The repeat masker annotated L1HS has a length of 6030 bp, starting at pos125? of canonical L1HS, on other occasions have seen this and determined it was a true L1HS. Would need some work to confirm in this case. I will add label Line-1  (it is INS:ME in latest VCF, May 25)  There is some uncertainty in this.

**Comment by jo-mc** [comment]
I wrote the wrong blat result and Line 1 position.   Reviewing I cant find a good source of this insertion. Possibly not reference and maybe in HG008-N? Puts more doubt on this being a true MEI without further study.
